src/includes/getting-started-primer/apple.mdx

```diff
@@ -8,6 +8,12 @@
 
 **Features:**
 
+- Multiple types of errors are captured, including:
+  - C++ Exception
+  - Mach Exceptions
+  - Unhandled Exceptions
+  - UNIX Signal
+  - Out of memory
```

Comment:
out of memory is still in Alpha - are we comfortable listing it here?
```suggestion
- Multiple types of errors are captured, including:
  - C++ exceptions
  - Mach exceptions
  - Unhandled exceptions
  - UNIX signal
  - Out of memory
```

Reply:
It's in beta now. We plan to GA next week.